all_repos/grep.py

```diff
@@ -86,12 +86,23 @@ def main(argv: Optional[Sequence[str]] = None) -> int:
     parser = argparse.ArgumentParser(
         description='Similar to a distributed `git grep ...`.',
         usage='%(prog)s [options] [GIT_GREP_OPTIONS]',
+        add_help=False,
+    )
+    # Handle --help like normal
+    parser.add_argument(
+        '--help', action='help', help='show this help message and exit',
     )
     cli.add_common_args(parser)
     cli.add_repos_with_matches_arg(parser)
     cli.add_output_paths_arg(parser)
     args, rest = parser.parse_known_args(argv)
 
+    # Handle -h like --help if it's the only remaining flag, otherwise defer
+    # handling to git grep
+    if rest == ['-h']:
+        parser.print_help()
+        return 0
```

Comment:
Question: Instead of doing this would you rather just always pass `-h` to `git grep`?

Reply:
yeah I would just remove this block